src/js/components/FormField/doc.js

```diff
@@ -34,6 +34,7 @@ export const doc = FormField => {
       of name, value, onChange (event => {}), while event has either event.value
       or event.target.value.`,
     ),
+    contentProps: PropTypes.object.description('Any valid Box prop.'),
```

Comment:
We should add on the description where would that be applied.

Reply:
Added more detail in the description about what they are applied to and may override